app-env.js

```diff
@@ -234,6 +234,18 @@ function getViewsPath() {
   return viewsPath;
 }
 
+function ensurePageTitle() {
+  if (!process.env[AppConstants.OCTOFARM_SITE_TITLE_KEY]) {
+    logger.info(
+        `~ ${AppConstants.OCTOFARM_SITE_TITLE_KEY} environment variable is not set. Assuming default: ${AppConstants.OCTOFARM_SITE_TITLE_KEY}=${AppConstants.defaultOctoFarmPageTitle}.`
+    );
+    printInstructionsURL();
```

Comment:
I would not log this message and would avoid printing instructions. I think this needs to be a silent setting just like prerelease (which is gonna be removed, but still).

Reply:
I'm sure I removed that but will do another push for it aha

Reply:
Ill do some pushups for too rapid rebasing!

Reply:
bahaha, I've lifted enough tables for the day does that count? 

Fixed anyway